src/infiniteGrid.js

```diff
@@ -287,12 +288,13 @@ eg.module("infiniteGrid", ["jQuery", eg, window, document], function($, ns, glob
 		_layoutItems: function(items) {
 			items.map(function(v) {
 				v.position = this._getItemLayoutPosition(v);
+				return v;
+			}, this).forEach(function(v) {
 				if (v.el) {
 					v.el.style.left = v.position.x + "px";
 					v.el.style.top = v.position.y + "px";
 				}
-				return v;
-			}, this);
+			});
 		},
```

Comment:
good. 
but, in order to improve performance, we should separate appying dom  actions.